bundles/org.openhab.ui/web/src/assets/item-types.js

```diff
@@ -1,6 +1,6 @@
 export const ItemTypes = [ 'Switch', 'Contact', 'String', 'Number', 'Dimmer', 'DateTime', 'Color', 'Image', 'Player', 'Location', 'Rollershutter', 'Group' ]
 export const GroupTypes = [ 'None', 'Switch', 'Contact', 'Number', 'Dimmer', 'Rollershutter', 'DateTime' ]
-export const Dimensions = [ 'Temperature', 'Pressure', 'Speed', 'Length', 'Angle', 'Intensity', 'Dimensionless', 'Acceleration', 'AmountOfSubstance', 'AngularAcceleration', 'AngularVelocity', 'Area', 'CatalyticActivity', 'DataAmount', 'DataRate', 'Duration', 'DynamicViscosity', 'ElectricCapacitance', 'ElectricCharge', 'ElectricConductance', 'ElectricCurrent', 'ElectricInductance', 'ElectricPotential', 'ElectricResistance', 'Energy', 'Force', 'Frequency', 'Illuminance', 'KinematicViscosity', 'LuminousFlux', 'LuminousIntensity', 'MagneticFlux', 'MagneticFluxDensity', 'Mass', 'MassFlowRate', 'Power', 'RadiationDoseAbsorbed', 'RadiationDoseEffective', 'RadioactiveActivity', 'SolidAngle', 'Torque', 'Velocity', 'Time', 'Volume', 'VolumetricDensity', 'VolumetricFlowRate' ]
+export const Dimensions = [ 'Area', 'Length', 'Mass', 'Pressure', 'Speed', 'Temperature', 'Volume', 'Dimensionless', 'Acceleration', 'AmountOfSubstance', 'Angle', 'ArealDensity', 'CatalyticActivity', 'DataAmount', 'DataTransferRate', 'Density', 'ElectricCapacitance', 'ElectricCharge', 'ElectricConductance', 'ElectricCurrent', 'ElectricInductance', 'ElectricPotential', 'ElectricResistance', 'Energy', 'Force', 'Frequency', 'Illuminance', 'Inensity', 'LuminousFlux', 'LuminousIntensity', 'MagneticFlux', 'MagneticFluxDensity', 'Power', 'RadiationDoseAbsorbed', 'RadiationDoseEffective', 'Radioactivity', 'SolidAngle', 'Time', 'VolumetricFlowRate' ]
```

Comment:
```suggestion
export const Dimensions = [ 'Area', 'Length', 'Mass', 'Pressure', 'Speed', 'Temperature', 'Volume', 'Dimensionless', 'Acceleration', 'AmountOfSubstance', 'Angle', 'ArealDensity', 'CatalyticActivity', 'DataAmount', 'DataTransferRate', 'Density', 'ElectricCapacitance', 'ElectricCharge', 'ElectricConductance', 'ElectricCurrent', 'ElectricInductance', 'ElectricPotential', 'ElectricResistance', 'Energy', 'Force', 'Frequency', 'Illuminance', 'Intensity', 'LuminousFlux', 'LuminousIntensity', 'MagneticFlux', 'MagneticFluxDensity', 'Power', 'RadiationDoseAbsorbed', 'RadiationDoseEffective', 'Radioactivity', 'SolidAngle', 'Time', 'VolumetricFlowRate' ]
```

Reply:
`Inensity` => `Intensity`
I think we'd better have those broken into one per line eventually - and maybe a way to keep them and others like the semantic tags current automatically... but preferably at build time, not with an API call.
But that's not urgent ;)

Reply:
Yes, a dynamical solution would make sense. We never could be sure which new dimensions we will have in the future.